dts/bindings/spi/spi-device.yaml

```diff
@@ -42,3 +42,9 @@ properties:
       enum:
         - 0
         - 32768
+    spi-cs-delay-us:
+       type: int
+       description: |
+         Additional delay, in microseconds not ticks, to add after asserting CS
```

Comment:
Is it necessary to say "not ticks"? It's called spi-cs-delay-us (which includes "-us") and the description already says "in microseconds". I think that's clear enough. Thanks!